libs/api/domains/national-registry-x-road/src/models/nationalRegistryChildGuardianship.model.ts

```diff
@@ -0,0 +1,13 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+
+@ObjectType()
+export class NationalRegistryXRoadChildGuardianship {
```

Comment:
FYI you could shorten the model name, so it is shorter when working with it in code. But then you would set the full name in the `@ObjectType` decorator.
```suggestion
@ObjectType('NationalRegistryXRoadChildGuardianship')
export class ChildGuardianship {
```